### PR TITLE
Add Aqua Container Security Platform

### PR DESCRIPTION
**Pull request by edvancecn**
Add aqua container security platform to rancher catalog.

### CODE REVIEW

**Comment by orangedeng** [comment]
Please combine all your commits into one. Thanks!

**Comment by edvancecn** [comment]
OK, it's done.

File: templates/aqua/0/rancher-compose.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.catalog:
+  name: "aqua"
+  version: "2.6.3"
+  description: "aqua csp 2.6.3"
+  minimum_rancher_version: v1.1.0
```
Comment (orangedeng) [inline templates/aqua/0/rancher-compose.yml]: I don't know if this catalog entry can work on v1.1.0. But it is better to use the Rancher version which you test this entry with.

Comment (edvancecn) [reply templates/aqua/0/rancher-compose.yml]: Thanks for reminder, I corrected commit title now. Also we have already tested aqua on rancher 1.1.0

**Comment by orangedeng** [comment]
LGMT

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**